shared/getting-started/getDeviceOnDash/var-som-mx6.md

```diff
@@ -0,0 +1,11 @@
+Currently balenaOS on the {{ $device.name }} is only capable of booting from SD card, but in the future [eMMC booting will be supported](https://github.com/balena-os/resin-imx6ul-var-dart/issues/64). To provision the device, do the following:
+1. Verify Switch SW5 is OFF (downwards).
+2. Set Boot select switch (SW6) right to boot from microSD Card.
+3. Push microSD card into the microSD card slot (J102) of the
+VAR-SOLOCustomBoard.
+4. Ensure the device has a connection to the internet.
+5. Switch ON (upwards) the SW5 switch
+
+While you wait, {{ $names.company.lower }} is partitioning your SD card, installing a custom Linux environment, and establishing a secure connection with our servers.
```

Comment:
```suggestion
While you wait, {{ $names.os.lower }} is partitioning your SD card, installing a custom Linux environment, and establishing a secure connection with our servers.
```

Reply:
resolved in https://github.com/balena-io/docs/pull/968/commits/4d24ac750c85be19ed8f4098780e6de9430c5277